src/GenerateCodeCoverageReports.ps1

```diff
@@ -0,0 +1,45 @@
+$root = Split-Path -Parent $PSScriptRoot
+$src = $PSScriptRoot
+
+$opencover = (Resolve-Path "$src\packages\OpenCover.*\tools\OpenCover.Console.exe" -ErrorAction Stop).ToString()
+$xunit = (Resolve-Path "$src\packages\xunit.runner.console.*\tools\xunit.console.exe" -ErrorAction Stop).ToString()
+$reportgenerator = (Resolve-Path "$src\packages\ReportGenerator.*\tools\ReportGenerator.exe" -ErrorAction Stop).ToString()
+$configuration = "Debug"
+
+if (Test-Path env:CONFIGURATION) {
+    $configuration = $env:CONFIGURATION
+}
+
+if (!(Test-Path "$root\CodeCoverage")) {
+	mkdir "$root\CodeCoverage"
+}
+
+Remove-Item "$root\CodeCoverage\*" -Recurse
+
+$testDlls = Get-ChildItem -Path $src -Filter "*.Tests" | % { """""$src\$_\bin\$configuration\$_.dll""""" }
```

Comment:
Not required since we are searching for test project folders at this point (no recurse flag) and use it to build the name to the dll.